release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -1309,6 +1313,63 @@ link:https://issues.redhat.com/browse/OCPBUGS-39428[(*OCPBUGS-39428*)]
+
+* Previously, in the technology preview oc-mirror plug-in v2 feature, when running behind a proxy with system proxy settings, oc-mirror did not use the system proxy configuration to recover signatures for releases. With this release, the system proxy settings are now applied during the signature recovery process. (link:https://issues.redhat.com/browse/OCPBUGS-37055[*OCPBUGS-37055*])
+
+* Previously, {product-title} release images were not visible in certain registries, such as {rhq-short}. This prevented users from installing {product-title} due to the missing release images. With this update, release images are always tagged to ensure they appear in registries like {rhq-short}, enabling proper installation. (link:https://issues.redhat.com/browse/OCPBUGS-36410[*OCPBUGS-36410*])
+
+* Previously, when using the technology preview oc-mirror plug-in v2 feature in Mirror-to-Disk mode, catalog images and contents were stored in `subfolders` under `working-dir`, based on the image digest. During the Disk-to-Mirror process in fully disconnected environments, oc-mirror tried to resolve the catalog image tag through the source registry, which was unavailable, leading to such errors:
+[source.terminal]
++
+----
+[ERROR] : [OperatorImageCollector] pinging container registry registry.redhat.io: Get "http://registry.redhat.io/v2/": dial tcp 23.217.255.152:80: i/o timeout
+----
++
+With this update, `oc-mirror` checks the local cache during the Disk-to-Mirror process to determine the digest, avoiding the need to query the registry. (link:https://issues.redhat.com/browse/OCPBUGS-36214[*OCPBUGS-36214*])
+
+* Previously, the technology preview oc-mirror plug-in v2 feature failed to mirror operator catalogs from disk-to-mirror in fully disconnected environments. This issue also affected catalogs that specified a `targetCatalog` in the `ImageSetConfiguration` file. With this update, oc-mirror v2 can now successfully mirror catalogs in fully disconnected environments, and the `targetCatalog` functionality works as expected. (link:https://issues.redhat.com/browse/OCPBUGS-34521[*OCPBUGS-34521*])
+
+* Previously, the `oc adm must-gather` command took a long time to gather CPU related performance data in large clusters. With this release, the data is gathered in parallel instead of sequentially, which shortens the data collection time. (link:https://issues.redhat.com/browse/OCPBUGS-34360[*OCPBUGS-34360*])
```

Comment:
```suggestion
* Previously, the `oc adm must-gather` command took a long time to gather CPU-related performance data in large clusters. With this release, the data is gathered in parallel instead of sequentially, which shortens the data collection time. (link:https://issues.redhat.com/browse/OCPBUGS-34360[*OCPBUGS-34360*])
```